pkg/tstune/tuner.go

```diff
@@ -489,12 +490,27 @@ func (t *Tuner) processTunables(totalMemory uint64, cpus int, quiet bool) error
 // processQuiet handles the iteractions when the user wants "quiet" output.
 func (t *Tuner) processQuiet(totalMemory uint64, cpus int) error {
 	t.handler.p.Statement(statementTunableIntro, parse.BytesToDecimalFormat(totalMemory), cpus)
-	if t.cfs.sharedLibResult == nil {
+
+	// Replace the print function with a version that counts how many times it
+	// is invoked so we can know whether to prompt the user or not. It doesn't
+	// make sense to ask for a yes/no if nothing would change.
+	count := uint64(0)
```

Comment:
maybe rename count to changedSettings to make it clearer what is counted

Reply:
done